From b7b3ae5475a8e3f167e78c32b4ab3f444e219c3a Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Thu, 10 Mar 2022 04:29:31 +0200
Subject: [PATCH] [SW-69351] rom: linker update for gaudi3 onix

Gaudi3's fpga environment has boot rom memory emulated. Hence
unlike older platforms, we do not need to run rom code on a
different memory in gaudi3. Hence this override is removed
for gaudi3.

Change-Id: Ib81f571758b205b3081322985de132f5e6351655
---

diff --git a/arch/gaudi3/arc/onix_arcboot_rom.lds b/arch/gaudi3/arc/onix_arcboot_rom.lds
index a013c5c..2085455 100644
--- a/arch/gaudi3/arc/onix_arcboot_rom.lds
+++ b/arch/gaudi3/arc/onix_arcboot_rom.lds
@@ -4,10 +4,3 @@
  */
 
 INCLUDE arcboot_rom.lds
-
-/* This is SRAM address, used for Onix targets */
-MEMORY
-{
-	/* TODO: Update address to run in onix env */
-	SBROM	: ORIGIN = 0x11000000, LENGTH = 64K
-}
